### ISSUE TITLE
translate one (or more) intertegular distance metrics into wikidata as a quantitative trait

### ISSUE THREAD

**Issue by jhpoelen** [Big-Bee-Network/trait-intertegular-distance#1]
Hi @Daniel-Mietchen ,

@seltmann et al. have prepared some example of intertegular distance measurements. This distance (as described in https://github.com/Big-Bee-Network/trait-intertegular-distance) is a proxy for body size/mass for bees. 

I was wondering whether you have any ideas on how to represent this metric in the wiki-verse. . . 

Looking forward to our session tomorrow!

**Comment by jhpoelen** [comment]
For examples see:

https://github.com/Big-Bee-Network/trait-intertegular-distance/blob/main/UCSB-Microscope.csv

and 

https://github.com/Big-Bee-Network/trait-intertegular-distance/blob/main/UCSB-NfN-2022-07-22.csv

**Comment by jhpoelen** [comment]
Oops, mistyped your handle @Daniel-Mietchen earlier. Apologies.

**Comment by jhpoelen** [comment]
@Daniel-Mietchen pointed out that it may be more practical to use a more generic trait like body size or body mass. Then, as a reference, point to the data publication. This data publication would be expected to include a description on how to the body size/mass estimate came about.

**Comment by Daniel-Mietchen** [comment]
By "practical", I mean doable with existing Wikidata properties or properties likely to get community approval.

**Comment by jhpoelen** [comment]
Here are some example of traits as listed in the biological traits project - 

https://www.wikidata.org/wiki/Wikidata:WikiProject_Biodiversity/Traits#Morphological_traits

**Comment by jhpoelen** [comment]
@Daniel-Mietchen suggesting to use some of the individuals of https://www.wikidata.org/wiki/Q179959 as a model for type specimen traits , a nearly extinct bird, with individuals documented on wiki data.

**Comment by jhpoelen** [comment]
We need to select an appropriate trait related to body size/mass available in wikidata .

See related discussions on various wikidata discussions - 

https://www.wikidata.org/wiki/Wikidata:Project_chat/Archive/2023/02#best_practice_for_the_representation_of_quantitative_data_and_their_metadata 

fyi @katjaschulz @jhammock - any thoughts on defining a body size/mass trait property in wikidata?

**Comment by jhpoelen** [comment]
@Daniel-Mietchen should we create a proposal to introduce body size as a wikidata property?

**Comment by jhpoelen** [comment]
@Daniel-Mietchen can you please share some examples of type specimen recorded in Wikidata?

**Comment by KatjaSchulz** [comment]
I agree with @Daniel-Mietchen that intertegular distance has little prospect of getting approved as a WikiData property. However, if you can develop a formula that converts intertegular distance measurements into estimates of body [length (P2043)](https://www.wikidata.org/wiki/Property:P2043) or [mass (P2067)](https://www.wikidata.org/wiki/Property:P2067), you could add statements for those properties to Wikidata with references to the publication where you present the intertegular distance value and to the paper where you explain the data conversion. You could also add a [nature of statement (P5102)](https://www.wikidata.org/wiki/Property:P5102) qualifier using something like [estimate (Q37113960)](https://www.wikidata.org/wiki/Q37113960) or [extrapolation (Q744069)](https://www.wikidata.org/wiki/Q744069) as the item. See here for available items for this qualifier: https://www.wikidata.org/wiki/Property:P5102#P2302

**Comment by Daniel-Mietchen** [comment]
> @Daniel-Mietchen can you please share some examples of type specimen recorded in Wikidata?

@jhpoelen @KatjaSchulz @seltmann @jhammock

Here is a [list of several hundred type specimens](https://query.wikidata.org/#%23title%3A%20Type%20specimens%0ASELECT%20DISTINCT%20%3Fitem%20%3FitemLabel%20%3Fstatements%20%3Fsitelinks%20WHERE%20%7B%0A%20%20%3Fitem%20wdt%3AP31%2Fwdt%3AP279%2a%20wd%3AQ51255340%20%3B%0A%20%20%20%20%20%20%20%20wikibase%3Astatements%20%3Fstatements%20%3B%0A%20%20%20%20%20%20%20%20wikibase%3Asitelinks%20%3Fsitelinks%20.%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%7D%0AORDER%20BY%20DESC%28%3Fsitelinks%29%20DESC%28%3Fstatements%29%0A). The links between them and the respective species are not modeled in a consistent fashion, so the query does not show the taxon names.

There are also [several thousand individuals from various taxa](https://query.wikidata.org/#%23title%3A%20Individuals%20of%20various%20taxa%0ASELECT%20DISTINCT%20%3Fitem%20%3FitemLabel%20%3Fvalue%20%3Ftaxonname%20%3Fstatements%20%3Fsitelinks%0A%7B%0A%20%20%3Fitem%20p%3AP10241%20%3Fstatement%20.%0A%20%20%3Fstatement%20ps%3AP10241%20%3Fdbvalue%20.%0A%20%20%3Fdbvalue%20wdt%3AP225%20%3Ftaxonname%20.%0A%20%20%3Fitem%20wikibase%3Astatements%20%3Fstatements%20%3B%0A%20%20%20%20%20%20%20%20wikibase%3Asitelinks%20%3Fsitelinks%20.%0A%20%20BIND%28IF%28wikibase%3AisSomeValue%28%3Fdbvalue%29%2C%20%22%22%2C%20%3Fdbvalue%29%20AS%20%3Fvalue%29%0A%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22%20%7D%0A%7D%0AORDER%20BY%20DESC%28%3Fsitelinks%29%20DESC%28%3Fstatements%29%0A)that have their own Wikidata entries.

**Comment by Daniel-Mietchen** [comment]
> @Daniel-Mietchen should we create a proposal to introduce body size as a wikidata property?

Yes, given the popularity of body size-related statements in the literature, this is worth exploring. As always, the devil here is in the details, e.g. in properly representing the link between the actual measurements and the inferred body size.

**Comment by seltmann** [comment]
Hi @jhpoelen @KatjaSchulz @Daniel-Mietchen the primary paper for conversion of intertegular span ([HAO:0002584](http://purl.obolibrary.org/obo/HAO_0002584)) is Cane, 1987 [https://www.jstor.org/stable/25084877](https://www.jstor.org/stable/25084877). 

Span is related to dry weight (body mass) by the equation y = 0.77(x)^0.405, where
 y is intertegular span and x is **dry weight**.

**Comment by jhpoelen** [comment]
@seltmann Thanks for the reference and I am excited by the prospect of having Big Bee contribute to wikidata and other trait datasets. 

@KatjaSchulz @jhammock would you like Big Bee to deliver some bee body mass measurements to EOL directly, or would you like to calculate them from intertegular span yourself. Are you still interested in trait datasets for EOL?

**Comment by jhammock** [comment]
This is what we've done in EOL in similar cases in other groups. Some are sourced from a single paper (which provided linear measurement and conversion formula), eg: https://eol.org/records/R556-PK219451427  

and some use separate sources for measurement and conversion method, eg: https://eol.org/records/R556-PK219453343

Our own data model is pretty crude, but for export to wikidata we might use "inferred from" the source of the measurement. I'm not sure what property is appropriate to associate the source of the conversion method, though...

**Comment by jhpoelen** [comment]
@jhammock thanks for sharing these example of converted datasets . I am hoping to motivate myself to come up with some way to follow your example for the intertegular distance-based dry mass estimates in collaboration with @seltmann with the goal to distribute as widely as practically possible. This would include the provenance/origin of the data and their associated authorities (e.g., people and their publications).
